src/sentry/status_checks/warnings.py

```diff
@@ -17,6 +21,10 @@ def check(self):
                         "issues" if len(self.__warning_set) > 1 else "issue",
                     ),
                     severity=Problem.SEVERITY_WARNING,
+                    # We need this manual URL building as this page is moved to react
+                    # and only the top-level entrypoint is defined and addressable in
+                    # our backend Django app.
+                    url=urljoin(reverse("sentry-admin-overview"), "status/warnings/"),
```

Comment:
@EvanPurkhiser @wedamija is this fine or should I just put a literal string here (feels worse to do that)

Reply:
nah this is better IMO

Reply:
Then setting up auto-merge, here we go... 🚀

Reply:
@EvanPurkhiser I wonder if we can make these links not treated as external links (which is the default now)? Is it safe to pass `ReactRouter` related stuff from here? I can pass it to `to=` but that feels unsafe.